scripts/suite_objects.py

```diff
@@ -1589,9 +1607,25 @@ def add_var_transform(self, var, compat_obj, vert_dim):
         from <var> to perform the transformation. Determine the indices needed
         for the transform and save for use during write stage"""
 
-        # Add dummy variable (<var>_local) needed for transformation.
-        dummy = var.clone(var.get_prop_value('local_name')+'_local')
-        self.__group.manage_variable(dummy)
+        # Add local variable (<var(local_name)>_local) needed for transformation.
+        # Do not let the Group manage this varaible. Handle local var
+        # when writing Group.
+        prop_dict = var.copy_prop_dict()
+        prop_dict['local_name'] = var.get_prop_value('local_name')+'_local'
+        # This is a local variable.
+        if 'intent' in prop_dict:
+            del prop_dict['intent']
+        # end if
+        local_trans_var = Var(prop_dict,
+                              ParseSource(_API_SOURCE_NAME,
+                                          _API_LOCAL_VAR_NAME, var.context),
+                              self.run_env)
+        found = self.__group.find_variable(source_var=local_trans_var, any_scope=False)
+        if not found:
+            lmsg = "Adding new local Group variable, '{}', for variable transform"
```

Comment:
That sounds weird, "local Group variable". Is it a variable specific for that scheme? It is still declared at the group level, or?

Reply:
Agreed. "local variable" is consistent. I will change.